dask/dataframe/groupby.py

```diff
@@ -1012,6 +1140,42 @@ def std(self, ddof=1, split_every=None, split_out=1):
         result = map_partitions(np.sqrt, v, meta=v)
         return result
 
+    @derived_from(pd.DataFrame)
+    def cov(self, ddof=1, split_every=None, split_out=1):
+        """Groupby covariance is accomplished by
```

Comment:
Just a note: this will show up in the public docstring. Did you want that? If so, Sphinx is probably going to complain that the number is off (the wrapped lines should start below the text, not the number).

Reply:
I think https://github.com/dask/dask/pull/4889/commits/bcc803fb879d40429ffa26847fa742ac87728d55 does what you are asking ?